index.php

```diff
@@ -121,11 +121,12 @@ function wdnInclude($path)
 							</div>
 							<div class="bp768-wdn-col-one-half">
 								<p class="col col-12 col-sm-8">
-									Resident appointments begin January 21st and run through April 8th. 
-									Appointments for International Student Scholars tax appointments begin March 6th and run 
-									through April 10th. Appointments are available at the following sites: Nebraska East Union, 
-									Anderson Library, and Jackie Gaughan Multicultural Center. To see the days and times for 
-									all the locations, please download site schedule. 
+									Resident appointments begin January 20th and run through April 11th. 
+									Tax Appointments for International Student Scholars begin March 5th and run 
+									through April 9th. Appointments are available at the following sites: Nebraska East Union, 
```

Comment:
Haha I'm not either--pure fluke I saw this error as I was going through the code base. Do you think the move is to pull these values from the database?

Reply:
If it will be the same in some way in some way from year to year, then we could just set it in the php. Otherwise, I think it will have to come from the database.